tests/console/suseconnect_scc.pm

```diff
@@ -41,6 +41,8 @@ sub run {
     if (is_transactional) {
         trup_call('register' . $cmd);
         trup_call('--continue run zypper --gpg-auto-import-keys refresh') if is_staging;
+        # Currently only micro6.0 support SL-Micro-Extras
```

Comment:
6.1 as  well, but there is SCC setup issue which is not sorted yet

Reply:
We need to wait till it is fixed.

Reply:
Update result for sle-micro6.1